posthog-core/src/index.ts

```diff
@@ -1076,7 +1076,13 @@ export abstract class PostHogCoreStateless {
     }
 
     const body = options.body ? options.body : ''
-    const reqByteLength = Buffer.byteLength(body, STRING_FORMAT)
+    let reqByteLength = -1
+    try {
+      reqByteLength = Buffer.byteLength(body, STRING_FORMAT)
+    } catch {
+      const encoded = new TextEncoder().encode(body)
+      reqByteLength = encoded.length
+    }
```

Comment:
unrelated to fixing it anyway
Buffer is node only

Reply:
Reported in https://github.com/PostHog/posthog-js-lite/issues/517, but because you didn't cut a new `posthog-web` version when you changed this, it's not live yet. Will release a new version tomorrow.

Reply:
Thanks for the fix :)